include/mbedtls/pk_internal.h

```diff
@@ -1,10 +1,12 @@
 /**
  * \file pk_internal.h
  *
- * \brief Public Key abstraction layer: wrapper functions
- */
-/*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ * \brief Public Key cryptography abstraction layer: built-in key types
+ *
+ *  This file contains built-in types for handling various key types using
+ *  the interface defined in pk_info.h.
+ *
```

Comment:
As in `pk_info.h`, end the Doxygen part here.

Reply:
Done